exposee-app/Routes/users.js

```diff
@@ -0,0 +1,86 @@
+import express from 'express';
+import bcrypt from 'bcrypt';
+import { User } from '../models/users.js';
+import { Op } from 'sequelize';
+
+const router = express.Router();
+
+
+router.get ('/user', async(req,res)=> {
+  
+  
+  const users = await User.findAll();
+  console.log(users)
+  res.status(200).json(users);
+  
+     });
+  
+   
+// Route for user registration
+router.post('/user', async (req, res) => {
+  const { username, password, email } = req.body;
+
+  try {
+    // Check if username or email already exists
```

Comment:
your test plan should ideally cover all lines of code, that means that you should have a test plan for checking if the username already exists, checking if the email already exists